util/constants/visitors.ts

```diff
@@ -15,6 +15,8 @@ export const ImpSpecVisitor: {ImportSpecifier: (path: Path)=> void} ={
     }
     // check to see if the property 'imported' is an identifier with the name 'Component'
     if (path.get('imported').isIdentifier({name: 'Component'})) {
+      console.log("path.node is:", path.node)
+      console.log('found an import statement')
```

Comment:
remove all console.log statements

Reply:
strange, this is from back at CS, will get rid of them.